CHANGELOG.md

```
@@ -1,36 +1,35 @@
   * Changed the `MxParseUtils` findByTags and findByPath methods to return the element values instead of the XML stream object
 
-#### 10.2.5 - January 2025
+#### 9.5.5 - January 2025
   * (PW-2113) `MxParseUtils` added a method to extract the enclosed MT from a multi-format MX message
   * (PW-2113) `MxParseUtils` added methods to extract comments from XML string
   * Added new SettlementInfo class, and added MxParseUtils#getSettlementInfo to extract it from a raw MX message.
-  * Moved and enhanced the MxSwiftMessage#findElement to support multiple element's path
+  * Moved and enhanced the MxSwiftMessage#findElement to support multiple element's path 
   * Update apache-commons-lang3 from 3.14.0 to 3.17.0 fixing derived apache-text dependency vulnerabilities
   * Gradle wrapper update to 8.12
+ 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)